Gemfile

```diff
@@ -64,8 +64,9 @@ gem 'rubyzip'
 # Stanford/Hydra related gems
 gem 'blacklight', '~> 6.0'
 gem 'blacklight-hierarchy'
+gem 'cocina-models', github: 'sul-dlss/cocina-models', branch: 'deserialize'
```

Comment:
I would rather wait for the gem to have its PR merged ...

Reply:
Agree.  I forgot that was why this was a draft and marked it "ready"